policies/lib/attestations.rego

```diff
@@ -19,3 +19,24 @@ taskrun_attestations := result {
 		att.predicate.buildType == taskrun_att_build_type
 	]
 }
+
+hacbs_test_task_result_name := "HACBS_TEST_OUTPUT"
+
+# (Don't call it test_results since test_ means a unit test)
+results_from_tests := [r |
+	att := pipelinerun_attestations[_]
+	task := att.predicate.buildConfig.tasks[_]
```

Comment:
Would it work to create a `tasks` variable similar to `pipelinerun_attestations`?